evm/evm-processor/src/ds-rpc/mapping.ts

```diff
@@ -64,15 +65,22 @@ function tryMapBlock(rpcBlock: RpcBlock, req: MappingRequest): Block {
     if (req.transactionList) {
         for (let i = 0; i < transactions.length; i++) {
             let stx = transactions[i]
-            let tx = new Transaction(header, i)
+            let isUnorderedChain = typeof stx == 'string' ? false : (!!stx.chainId && !!stx.transactionIndex && UNORDERED_TRANSACTION_CHAINS.includes(stx.chainId))
```

Comment:
Since these variables don't get redefined within this scope, let's use `const` instead. I'm surprised the typescript compilation does not error out